### PR TITLE
MODLD-381: Convert MARC field 050 to BF Lite Classification resource

### PR DESCRIPTION
**Pull request by askhat-abishev**

### CODE REVIEW

File: src/main/resources/marc4bibframe.yml [hunk 1]
```diff
@@ -33,13 +41,6 @@ fieldRules:
                 data:
                 expression: args.contains(data.substring(22, 23))
                 args: a, b, c, d, e, f, g, j
-        - types: CATEGORY_SET
```
Comment (askhat-abishev) [inline src/main/resources/marc4bibframe.yml:36]: these changes that were made in scope of MODLD-368 are incorrect (don't know why tests were passing)
in this PR I've placed the configuration in the correct place (line 23)

File: src/main/java/org/folio/marc4ld/service/ld2marc/mapper/impl/classification/Ld2MarcLcClassificationMapper.java [hunk 2]
```diff
@@ -1,78 +1,42 @@
...
-        ind1 = ZERO;
-      } else if (statuses.contains(NUBA)) {
-        ind1 = ONE;
-      }
+    if (hasLinkInEdge(resource, STATUS, UBA)) {
+      ind1 = ZERO;
+    } else if (hasLinkInEdge(resource, STATUS, NUBA)) {
+      ind1 = ONE;
     }
     return ind1;
```
Comment (aleksei-pronichev) [inline src/main/java/org/folio/marc4ld/service/ld2marc/mapper/impl/classification/Ld2MarcLcClassificationMapper.java:35]: minor, just simplify
```java
    if (hasLinkInEdge(resource, STATUS, UBA)) {
      return ZERO;
    } else if (hasLinkInEdge(resource, STATUS, NUBA)) {
    return ONE;
    }
    return SPACE;
```

**Review by aleksei-pronichev — APPROVED** [review]

File: src/main/resources/marc4bibframe.yml (hunk 1 above)
Comment (pkjacob) [reply src/main/resources/marc4bibframe.yml:36]: 👍 . Thank You for fixing this.

That is strange. I am also not understanding how the tests were passing earlier. Do you know @aleksei-pronichev ?

**Review by pkjacob — APPROVED** [review]

File: src/main/resources/marc4bibframe.yml (hunk 1 above)
Comment (aleksei-pronichev) [reply src/main/resources/marc4bibframe.yml:36]: need to recheck test cases for this situation. (suggest to creat little tech debt)

**Pull request merged**